### PR TITLE
Add  Automate modeling for Embedded Ansible Events.

### PR DESCRIPTION
**Pull request by lfu**
https://www.pivotaltracker.com/story/show/140758691

### CODE REVIEW

**Comment by lfu** [comment]
@miq-bot assign @gmcculloug
@miq-bot add_label providers/ansible_tower,automate,euwe/no

**Comment by lfu** [comment]
@miq-bot add_label providers/ansible_tower
@miq-bot add_label automate

File: content/automate/ManageIQ/System/Event/EmsEvent/EMBEDDED_ANSIBLE.class/__class__.yaml [hunk 1]
```diff
@@ -0,0 +1,433 @@
+---
+object_type: class
+version: 1.0
+object:
+  attributes:
+    description: 
+    display_name: Ansible Tower
+    name: ANSIBLE_TOWER
```
Comment (gmcculloug) [inline content/automate/ManageIQ/System/Event/EmsEvent/EMBEDDED_ANSIBLE.class/__class__.yaml]: Should be:
```yaml
  display_name: Ansible
  name: Embedded Ansible
```

Comment (lfu) [inline content/automate/ManageIQ/System/Event/EmsEvent/EMBEDDED_ANSIBLE.class/__class__.yaml]: 👍 Right.
Should we use ```Embedded Ansible``` to tell from external Ansible?
```
    display_name: Embedded Ansible
    name: EMBEDDED_ANSIBLE
```

Comment (gmcculloug) [inline content/automate/ManageIQ/System/Event/EmsEvent/EMBEDDED_ANSIBLE.class/__class__.yaml]: I do not think we need to duplicate the information.  Can we use this:
```yaml
  display_name:
  name: Embedded_Ansible
```

The duplications and upper-case make it look odd
image

I would prefer to just have:
image

As for external Ansible that is distinguished by the "Ansible_Tower" name.

**Comment by bzwei** [comment]
I agree we don't need to do refresh on any embedded tower events.
If we must enable the refresh, two scenarios need to be considered:
1. Refresh does not include job at all.
2. ManageIQ playbook service creates/deletes inventory before/after execution. Running refreshes to collect a temporary inventory and delete it later makes no sense.

**Pull request merged**